client/src/store/index.js

```diff
@@ -56,9 +56,19 @@ const Store = types
       }
     })
 
+    const setOrganizationalBranding = flow(function* (diagnosisMessage, exposureMessage) {
```

Comment:
This properties should be saved to the db. Hit me up in Slack and I can show you how.

Reply:
ah I assumed there was a hidden setter for the properties.